pandas/core/generic.py

```diff
@@ -4688,6 +4688,31 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         Returns
         -------
         wh : same type as caller
+
+        Notes
+        -----
+        The signature for :func:`DataFrame.where` differs from
+        :func:`numpy.where`. Roughly ``df1.where(m, df2)`` is equivalent to
+        ``np.where(m, df1, df2)``.
+
+        Examples
```

Comment:
need a See Also section that references Mask (ideally also examples there as well!)